Extension/src/LanguageServer/client.ts

```diff
@@ -475,6 +476,9 @@ class DefaultClient implements Client {
             if (changedSettings["clang_format_path"]) {
                 let settings: CppSettings = new CppSettings(this.RootUri);
                 this.languageClient.sendNotification(UpdateClangFormatPathNotification, util.resolveVariables(settings.clangFormatPath, this.AdditionalEnvironment));
+            } else if (changedSettings["intelliSenseCachePath"]) {
```

Comment:
I don't think it should be `else if `. You can change multiple things at once, right?

Reply:
Yeah, good catch.